projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.prod.json

```diff
@@ -5,8 +5,7 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "full",
-    "enableIvy": true,
+    "compilationMode": "partial",
```

Comment:
Just a note:  
 - The packaging using Ivy and this compilation mode were set in this manner to resolve loading issues while consuming the library in the end application.
 Just to have in mind, if the consuming project have troubles, to look into this.

Reply:
The publishing in dry-run using full Ivy mode was successful https://gitlab.com/vmware-analytics/versatile-data-kit/-/pipelines/797108389, closing the conversation.

Last time, the npm registry discarded the package compiled in --dry-run for full Ivy mode. Rolled back the full Ivy mode, keeping less diverging as passible for easier adoption back.